Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/ModularViewportCameraController.cpp

```diff
@@ -172,13 +172,11 @@ namespace AtomToolsFramework
         controller->SetupCameraControllerPriority(m_priorityFn);
         controller->SetupCameraControllerViewportContext(m_modularCameraViewportContext);
 
-        auto handleCameraChange = [this]([[maybe_unused]] const AZ::Matrix4x4& cameraView)
+        auto handleCameraChangeFn = [this]([[maybe_unused]] const AZ::Matrix4x4& cameraView)
```

Comment:
The _Fn_ stood out as odd, but looks like it's the standard for this file (m_priorityFn).

Reply:
That's a fair point... though we do use it in a few places, I'll have a think if it makes sense to keep or not, thanks for the suggestion! 👍